client/src/components/layouts/dashboardLayout/DashboardLayout.js

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+import Sidebar from '../layoutComponents/Sidebar';
+import Header from '../layoutComponents/Header';
+import { Container, Col, Row } from 'react-bootstrap';
+import SidebarData from '../SidebarData';
```

Comment:
We can probably rename this to DashboardSidebarData to keep consistency with EditorSidebarData naming convention.

Reply:
Make sense